app/src/main/res/layout/activity_main.xml

```diff
@@ -4,59 +4,163 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Let's change all EditText views into TextInputLayouts

Reply:
yes please will do that